src/open_inwoner/openzaak/tests/test_cases.py

```diff
@@ -742,14 +742,18 @@ def test_list_cases_logs_displayed_case_ids(self, m):
         logs = list(TimelineLog.objects.all())
 
         case_log = [
-            l for l in logs if self.zaak1["identificatie"] in l.extra_data["message"]
+            log
+            for log in logs
+            if self.zaak1["identificatie"] in log.extra_data["message"]
```

Comment:
Maybe not the best example but I want to be able to use single letter variables in cases like this.